rules/macos/persistence_screensaver_plist_file_modification.toml

```diff
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2021/10/05"
+maturity = "production"
+updated_date = "2021/10/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a screensaver plist file is modified by an unexpected process. An adversary can maintain persistence on
+a macOS endpoint by creating a malicious screensaver (.saver) file and configuring the screensaver plist file to execute
+code each time the screensaver is activated.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Screensaver Plist File Modified by Unexpected Process"
+note = """## Triage and analysis
+
+- Analyze the plist file modification event to identify whether the change was expected or not
+- Investigate the process that modified the plist file for malicious code or other suspicious behavior
+- Identify if any suspicious or known malicious screensaver (.saver) files were recently written to or modified on the host"""
+references = [
+    "https://posts.specterops.io/saving-your-access-d562bf5bf90b",
+    "https://github.com/D00MFist/PersistentJXA",
+]
+risk_score = 47
+rule_id = "e6e8912f-283f-4d0d-8442-e0dcaf49944b"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Persistence"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where event.type != "deletion" and
+  file.name: "com.apple.screensaver.*.plist" and
+  file.path : (
+    "/Users/*/Library/Preferences/ByHost/*",
+    "/Library/Managed Preferences/*",
+    "/System/Library/Preferences/*"
+    ) and
+  /* Filter OS processes modifying screensaver plist files */
+  not process.executable : (
```

Comment:
I am seeing some management software like jamf as well as 3rd party disk management tools touch this - even launchd in some instances. So we can either approach it with an exclusion list like this or using an inclusion list on suspicious processes like shells, osascript, cp, mv, etc.

I think either approach works based on your observations. It is refined to a point that users can add specific exclusions

Reply:
At this point it is probably negligible enough to ignore. Maybe a better fit for user level exclusions anyway